projects/netgrif-components-core/src/lib/navigation/service/uri.service.spec.ts

```diff
@@ -18,42 +21,100 @@ describe('UriService', () => {
         expect(service).toBeTruthy();
     });
 
+    it('should get root node', () => {
+        const root = service.root;
+        expect(root.id).toEqual('root');
+        expect(service.isRoot(root)).toBeTruthy();
+        expect(service.activeNode).toEqual(root);
+    });
+
+    it('should set and reset activeNode', done => {
+        service.getNodeByPath(MockUriResourceService.TEST1_PATH).subscribe(node => {
+            service.activeNode = node;
+            expect(service.activeNode).not.toEqual(service.root);
+            service.reset();
+            expect(service.activeNode).toEqual(service.root);
+            done();
+        });
+    });
+
     it('should get node by path', done => {
-        const sub = service.getNodeByPath('test');
-        sub.subscribe(res => {
-            expect(res.uriPath).toEqual('root/test1')
+        service.getNodeByPath(MockUriResourceService.TEST1_PATH).subscribe(res => {
+            expect(res.uriPath).toEqual(MockUriResourceService.TEST1_PATH);
+            expect(res.id).toEqual(MockUriResourceService.TEST1_ID);
             done();
-        })
+        });
+    });
+
+    it('should get child nodes', done => {
+        service.getChildNodes(service.root).subscribe(res => {
+            expect(res.length).toEqual(2);
+            res.forEach(r => {
+                expect(r.id).toContain('test');
+                expect(r.parentId).toEqual('root');
+            });
+            done();
+        });
+    });
+
+    it('should get cases of node', done => {
+        service.getCasesOfNode().subscribe(page => {
+            expect(page.content.length).toEqual(1);
+        });
+        done();
+    });
+
+    it('should get siblings of a node', done => {
+        service.getNodeByPath(MockUriResourceService.TEST1_PATH).subscribe(node => {
+            expect(node.id).toEqual(MockUriResourceService.TEST1_ID);
+            service.getSiblingsOfNode(node).subscribe(siblings => {
+                expect(siblings.find(n => n.id === MockUriResourceService.TEST2_ID)).not.toBeUndefined();
+                done();
+            });
+        });
     });
 
     it('should get nodes by level', done => {
-        const sub = service.getNodesOnLevel(1);
-        sub.subscribe(res => {
-            expect(res.length).toEqual(2)
+        service.getNodesOnLevel(1).subscribe(res => {
+            expect(res.length).toEqual(2);
             done();
-        })
+        });
     });
 
-    it('should get child nodes', done => {
-        const sub = service.getChildNodes();
-        sub.subscribe(res => {
-            expect(res.length).toEqual(2)
+    it('it should return path of a node\'s parent', done => {
+        service.getNodeByPath(MockUriResourceService.TEST1_PATH).subscribe(node => {
+            const parentPath = service.resolveParentPath(node);
+            expect(parentPath).toEqual('root');
             done();
-        })
+        });
     });
 
+    /*it('should get parts of path', done => {
```

Comment:
why not just remove the test?

Reply:
Removed. Sorry, it was a remnant from another branch.